python/ray/tests/test_memory_pressure.py

```diff
@@ -438,5 +439,30 @@ def test_put_object_task_usage_slightly_below_limit_does_not_crash():
         )
 
 
+@pytest.mark.skipif(
+    sys.platform != "linux" and sys.platform != "linux2",
+    reason="memory monitor only on linux currently",
+)
+def test_legacy_memory_monitor_disabled_by_oom_killer():
+    os.environ["RAY_MEMORY_MONITOR_ERROR_THRESHOLD"] = "0.5"
+    with ray.init(
+        _system_config={
+            "memory_monitor_interval_ms": 50,
```

Comment:
can you double check if this fails when you don't update the function `disabled`?

Reply:
yes it fails